docs/internet-identity-spec.adoc

```diff
@@ -536,6 +539,23 @@ This flow is the boring default
 +
 (This could include a button to go to the management screen, or maybe this _is_ the management screen with a info box.)
 
+=== Flow: Setup recovery
+1. The user is offered two options for recovery
+  * A security key
+  * A BIP-39 seed phrase
+  * (Skip?)
+2. Depending on their choice
+ a) If they choose the seed phrase it is generated and displayed to them with a copy button and an explanation of having to keep this phrase secure
+ b) 👆 If they choose the security key they generate new credentials for the touched device
+3. The device is added to the users account with the `is_recovery` flag set
```

Comment:
Would they be asked for an alias?

Reply:
If we want to expand the scope of this ticket, I would solve this by creating two ways of adding (normal, non-recovery) devices:

 * Add this laptop/phone as an authentication option.
 * Add a portable security device as an authentication option.

The only effect is that we set the `cross-platform` or the other on creation, else we do the same thing. Maybe store this choice with the device data, so that we can maybe show a suitable icon.

The benefit is that user’s have more guidance and more control, and less OS-specific dialogs.

And once we have that, we can allow users to designate their devices as recovery devices, by only allowing that for cross-platform devices.